### PR TITLE
BD-1317: Add article for email performance dashboard

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-1317**

#### Is this change associated with a Braze feature/product release?
- [ x] Yes (**4/5/22**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/your_analytics_dashboards/email_performance_dashboard.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+nav_title: Email Performance Dashboard
+article_title: Email Performance Dashboard
+page_order: 2
+page_type: reference
+description: "This reference article..."
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/your_analytics_dashboards/email_performance_dashboard.md:6]:
```suggestion
description: "This reference article covers the Email Performance Dashboard, which allows you to view performance metrics for your entire email channel from both campaigns and Canvases."
```

**Comment by bre-fitzgerald** [comment]
Hey @wendy-p, this is ready for your review! Just a heads up, I added an image of the dashboard as a placeholder, I'll replace that with a final version before publishing :)

File: _docs/_user_guide/data_and_analytics/your_analytics_dashboards/email_performance_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+#### If a metric displays "0%" or "0"
+
+This means Braze recorded zero for that particular metric during the time frame you’ve selected.
+
+#### If a metric displays "N/A"
+
+This means that while Braze recorded positive counts for a particular metric for the time frame you’ve selected, the denominator for the rate calculation (either sends or deliveries in most cases) was zero. This can occur when emails are sent out on one day and opens and clicks are recorded the following days, if your selected time frame does not include the date the emails were sent.
+
+#### If a metric displays “--”
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/your_analytics_dashboards/email_performance_dashboard.md:50]: @wendy-p Just wanted to call out something out here—I've been doing a lot of research on accessibility as part of some ongoing Beacon work, and I believe using "--" might cause issues with screenreaders (specifically NVDA won't voice anything at all). If possible at this stage (or in a v2?), I'd suggest changing this to something like "No data".

File: _docs/_user_guide/data_and_analytics/your_analytics_dashboards.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+guide_top_text: "This section is home to articles on the analytics dashboards available within Braze."
+
+page_type: landing
+description: "This landing page is home to articles on the analytics dashboards available within Braze."
+tool: Reports
+
+guide_featured_title: "Section Articles"
+guide_featured_list:
+  - name: Overview Data
+    link: /docs/user_guide/data_and_analytics/your_analytics_dashboard/understanding_your_app_usage_data/
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/your_analytics_dashboards.md:16]:
```suggestion
    link: /docs/user_guide/data_and_analytics/your_analytics_dashboards/understanding_your_app_usage_data/
```

File: _docs/_user_guide/data_and_analytics/your_analytics_dashboards/email_performance_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+## Metrics calculations
+
+The calculations for different metrics in the email performance dashboard is the same as those on an individual message level (i.e., Campaign Analytics). On this dashboard, the metrics are aggregated across all campaigns and Canvases for the date range you’ve selected. 
+
+To learn more about these definitions, refer to [Email metrics]({{site.baseurl}}/user_guide/message_building_by_channel/email/reporting_and_analytics/email_reporting#email-metrics).
+
+| Metric | Type | Calculation |
+| --- | --- | ---- |
+| Sends | Count | Total number of sends across each day in the date range |
+| Delivery Rate | Rate | (Total number of deliveries across each day in the date range) / (Total number of sends across each day in the date range) |
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/your_analytics_dashboards/email_performance_dashboard.md:29]:
```suggestion
| Delivery rate | Rate | (Total number of deliveries across each day in the date range) / (Total number of sends across each day in the date range) |
```

**Review by KellieHawks — APPROVED** [review]
Looks great!

**Pull request merged**